OxfordBodleian/Juel-Jensen/BDLaethg45.xml

```diff
@@ -37,10 +37,81 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                     <msContents>
                         <summary/>
                         
-                        <!--<msItem xml:id="ms_i1">
-                      
-                        </msItem>-->
+                        <msItem xml:id="ms_i1">
+                      <locus from="1r1" to="1r21"/>
+                                  <title type="complete" ref="LIT6714PPIllness"/>
+                            <textLang mainLang="gez"/>
+                            <incipit xml:lang="gez"><hi rend="rubric">በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩ አምላክ፡ ስቡሕ፡ ወውዱስ፡ ዘሣረረ፡ ኵሎ፡ ዓለም፡
+                                በ</hi>አሐቲ፡ ቃል፡ ሃሌ፡ ሉያ፡ እ<hi rend="ligature">ግዚ</hi>ኦ፡ መሐረነ፡ እ<hi rend="ligature">ግዚ</hi>ኦ፡ መሐከነ፡ 
+                                እ<hi rend="ligature">ግዚ</hi>ኦ፡ ተሠሃለነ፡ ስብሐት፡ ለአብ፡ ስብሐት፡ ለወልድ፡ ስብሐት፡ ለመንፈስ፡ ቅዱስ፡
+                                ቅዱ<hi rend="rubric">ስ፡ ቅዱስ፡ ቅዱስ፡ እ<hi rend="ligature">ግዚ</hi>አብሔር፡ አምላከ፡ አማልክት፡</hi></incipit>
+                            <explicit xml:lang="gez">ወእምኵሉ፡ ደዌ፡ ወሕማም፡ አድኅና፡ ለአመትከ፡ ወለተ፡ <space reason="rubrication"/>
+                            <hi rend="rubric">ወገጽ፡ ዘፍፁም፡ በጽልመት፡ ፈርሃ፡ ወደንገፀ፡</hi> ዲያብሎስ፡ ርእዮ፡ ብሁተ፡ ልደት፡ በሥጋ፡ አምላክ፡ በሲኦል፡ በዝ፡ ቃለ፡ መለኮትከ፡
+                                አድኅና፡ ለአመትከ፡ <persName ref="PRS13850WalattaS" role="owner">ወለተ፡ <hi rend="rubric">ሰንበት፡ ትበርህ፡</hi></persName></explicit>
+                        </msItem>
                         
+                        <msItem xml:id="ms_i2">
+                            <locus from="1r21" to="1r30"/>
+                               <title type="complete" ref="LIT6715PPQwerdat"/>
+                            <textLang mainLang="gez"/>
+                            <!--this is the complete text-->
+                            <incipit xml:lang="gez"><hi rend="rubric">ጸሎት፡</hi> በእንተ፡ ሕማመ፡ ቍርፀት፡ ኪስ፡ ኪስ፡ ኪስ፡ አራኪስ፡  ስርንኪስ፡ ስርዋጽ፡ 
+                            ምርዋጽ፡ ስር፡ ነኬር፡ ሲቃ፡ ወጼቃ፡ ወቤቓ፡ ድማኄል፡ አዶናይ፡ ቄጼቤ፡ ቄጼቤ፡ ቄጼቤ፡</incipit>
+                            <explicit xml:lang="gez">ቄፌኑ፡ ቄፌኑ፡ ቄፌኑ፡ ቡያክ፡ ቡያክ፡ ሞሊ፡ ሞሊ፡ ሞሊ፡ ኪያሞሌ፡ ኪያሞሌ፡ ኪያሞሌ፡ ዘአርጋእክ፡ ኃይለ፡
+                                በረድ፡ ወነፋስ፡ ከማሁ፡ አርግዕ፡ ወአቁም፡ ሕማመ፡ ቁርፀት፡ እምላእለ፡ አመትከ፡ <persName ref="PRS13850WalattaS">ወለተ፡ <hi rend="rubric">ሰንበት፡ ወለተ፡ ትበርሂ፡</hi></persName></explicit>
+                        </msItem>
+                        
+                        <msItem xml:id="ms_i3">
+                            <locus from="1r30" to="1r39"/>
+                            <title type="complete" ref="LIT6716PPWegat"/>
+                            <textLang mainLang="gez"/>
+                            <!--this is the complete text-->
```

Comment:
Concerning the solution of splitting the entire text into incipit and explicit: from a certain point of view I think it would be preferable to indicate that the text provided is the whole one (as in the work records), but if there is no way to encode the entire text in this position with one single element (and more importantly have it printed), then we must necessarily resort to incipit/explicit.

Reply:
I've thought it over and included the complete prayers additionally in the div edition (where they belong), and deleted the explicits and put the entire text into incipit. I think then it will be relatively easy to find a rule in these cases to print instead of "incipit" "complete text" or something similar. I hope that works as a practical solution for the time being.